drivers/builtin/src/pk.c

```diff
@@ -204,7 +204,7 @@ int mbedtls_pk_can_do(const mbedtls_pk_context *ctx, mbedtls_pk_type_t type)
         return 0;
     }
 
-    return ctx->pk_info->can_do(type);
+    return ctx->pk_info->can_do((mbedtls_pk_type_t) type);
```

Comment:
`type` already has the type `mbedtls_pk_type_t` so this cast does nothing. `mbedtls_pk_can_do` and the underlying `can_do` methods are based on `mbedtls_pk_type_t`, so I'm not sure what there would be to do here.